spanner/google/cloud/spanner/database.py

```diff
@@ -389,8 +389,7 @@ def batch(self):
         """
         return BatchCheckout(self)
 
-    def snapshot(self, read_timestamp=None, min_read_timestamp=None,
-                 max_staleness=None, exact_staleness=None):
+    def snapshot(self, **kw):
```

Comment:
We also use `kwargs` 1065 times in this codebase. Also, we *do use pylint* (even though it's temporarily in warning-only mode). Our [styleguide](https://google.github.io/styleguide/pyguide.html) doesn't explicit call this out, but the one example that uses key-word args uses `kwargs`. Why add more to the fire? Please just use `kwargs`, It's such a minor thing. We can address the others when we decide to re-enable pylint in error mode.